src/middlewares/logging.ts

```diff
@@ -0,0 +1,68 @@
+import express from "express";
+
+class Logging {
+	RequestResponseLogger = (data: any, req:express.Request, res:express.Response, _next:express.NextFunction) => {
```

Comment:
Can we use `unknown` instead of `any` please? Although we don't like `unknown` type but I guess this is inevitable in this case.

Reply:
The only reason is using latest `typescript` syntax and types.